Move the technical docs to serverless-manager (#166)
kyma-project/serverless-manager: docs/user/README.md

"Serverless" refers to an architecture in which the infrastructure of your applications is managed by cloud providers. Contrary to its name, a serverless application does require a server but it doesn't require you to run and manage it on your own. Instead, you subscribe to a given cloud provider, such as AWS, Azure, or GCP, and pay a subscription fee only for the resources you actually use. Because the resource allocation can be dynamic and depends on your current needs, the serverless model is particularly cost-effective when you want to implement a certain logic that is triggered on demand. Simply, you get your things done and don't pay for the infrastructure that stays idle.

> Kyma Serverless Module offers a kubernetes-based platform on which you can build, run, and manage serverless applications in Kubernetes. These applications are called **Functions** and they are based on the[Function custom resource (CR)](/docs/user/06-10-function.md) objects. They contain simple code snippets that implement a specific business logic. For example, you can define that you want to use a Function as a proxy that saves all incoming event details to an external database.

Reviewer: ```suggestion
Kyma Serverless Module offers a Kubernetes-based platform on which you can build, run, and manage serverless applications in Kubernetes. These applications are called **Functions**, and they are based on the [Function custom resource (CR)](/docs/user/06-10-function.md) objects. They contain simple code snippets that implement specific business logic. For example, you can define that you want to use a Function as a proxy that saves all incoming event details to an external database.
```

> Kyma offers a service (known as "functions-as-a-service" or "FaaS") that provides a platform on which you can build, run, and manage serverless applications in Kubernetes. These applications are called **Functions** and they are based on the[Function custom resource (CR)](/docs/user/06-10-function.md) objects. They contain simple code snippets that implement a specific business logic. For example, you can define that you want to use a Function as a proxy that saves all incoming event details to an external database.

Reviewer 2: ```suggestion
Kyma Serverless Module offers a kubernetes-based platform on which you can build, run, and manage serverless applications in Kubernetes. These applications are called **Functions** and they are based on the[Function custom resource (CR)](/docs/user/06-10-function.md) objects. They contain simple code snippets that implement a specific business logic. For example, you can define that you want to use a Function as a proxy that saves all incoming event details to an external database.
```

## Useful links

> If you want to perform some simple and more advanced tasks, check [Serverless tutorials](/docs/user/00-01-tutorials.md).

Reviewer: ```suggestion
If you want to perform some simple and more advanced tasks, check the [Serverless tutorials](/docs/user/00-01-tutorials.md).
```
